extensions/amp-list/0.1/amp-list.css

```diff
@@ -80,6 +73,10 @@ amp-list[load-more] [load-more-loading].i-amphtml-default-ui .i-amphtml-list-loa
   border-radius: 24px;
 }
 
+[load-more] div[role="list"] {
```

Comment:
We could restrict this to only `default-ui`, since it's possible for people to want an overlaid UI, and they can just override this.